gobblin-service/src/main/java/org/apache/gobblin/service/modules/flowgraph/pathfinder/AbstractPathFinder.java

```diff
@@ -77,18 +77,20 @@
   protected Long flowExecutionId;
   protected FlowSpec flowSpec;
   protected Config flowConfig;
+  protected Map<String, String> dataNodeAliasMap;
 
-  AbstractPathFinder(FlowGraph flowGraph, FlowSpec flowSpec)
+  AbstractPathFinder(FlowGraph flowGraph, FlowSpec flowSpec, Map<String, String> dataNodeAliasMap)
       throws ReflectiveOperationException {
     this.flowGraph = flowGraph;
     this.flowSpec = flowSpec;
     this.flowExecutionId = FlowUtils.getOrCreateFlowExecutionId(flowSpec);
     this.flowConfig = flowSpec.getConfig().withValue(ConfigurationKeys.FLOW_EXECUTION_ID_KEY, ConfigValueFactory.fromAnyRef(flowExecutionId));
+    this.dataNodeAliasMap = dataNodeAliasMap;
```

Comment:
no big deal to store if you wish, but do you need a field?  seems it's only passed through invocation args, prepared during this same ctor (that looks like good call BTW)